c4p2n/extractors.py

```diff
@@ -2,25 +2,29 @@
 
 
 def get_answer_type(answer: Dict[str, Any]) -> str:
-    return answer["type"]
+    return answer["type"]  # type: ignore  # will be fixed with #2
 
 
 def field_ref(answer: Dict[str, Any]) -> str:
-    return answer["field"]["ref"]
+    return answer["field"]["ref"]  # type: ignore  # will be fixed with #2
 
 
 def extract_text(text_answer: Dict[str, Any]) -> str:
-    return text_answer["text"]
+    return text_answer["text"]  # type: ignore  # will be fixed with #2
 
 
 def extract_choices(choices_answer: Dict[str, Any]) -> List[str]:
-    return choices_answer["choices"]["labels"]
+    return choices_answer["choices"]["labels"]  # type: ignore  # will be fixed with #2
 
 
 answer_type_to_extractor = {"text": extract_text, "choices": extract_choices}
 
 
 def extract_answer(answer: Dict[str, Any]) -> Union[str, List[str]]:
     answer_type = get_answer_type(answer)
-    extractor = answer_type_to_extractor[answer_type]
-    return extractor(answer)
+    if answer_type == "text":
+        return extract_text(answer)
+    elif answer_type == "choices":
+        return extract_choices(answer)
+    else:
+        raise TypeError(f"Unexpected answer type: {answer_type}")
```

Comment:
Do we really now need `answer_type_to_extractor` then? I myself would prefer to use a mapping-style here but this is also OK if it does its job and satisfies mypy checks.

Reply:
Well, actually I guess this is not that important.